### DISCUSSION TITLE
Newbie here...

### DISCUSSION THREAD

**Discussion by ReaLKing** [iptv-org/community#198] [Show and tell]
Hello all, 

Newbie here- was just invited over and looking for some possible solution or any assistance if someone may have the answer.

Example- I am running a channels.dvr server off one of my Nvidia shields and want to add more m3u8's or playlists, or even would like to stream content to my site vs Kodi I can only do so in http/https/rtmp format. 

The issue I face is when I pull a link as follows:
plugin://plugin.video.streamlink-tester/?action=play&url=http://totalhdsport.xyz/hd/10.php
or 
plugin://plugin.video.tvone1112/play/2104/play.pvr

Is there a method of streamlnk I can use, or maybe a different way of possibly integrating FFMPEG to change the way its sent out.

Any info would be greatly appreciated!  Look forward to contribute where I can.

**Comment by Sphinxroot** [comment]
@LaneSh4d0w  any clue about this ?

this play well on kodi ,but when i try any online streamlinker  it wont
plugin://plugin.video.streamlink-tester/?action=play&url=http://totalhdsport.xyz/hd/10.php

**Reply by BellezaEmporium** [reply]
Something seems quite fishy with this one, knowing that it gives the link on a Base64 via a script from a weirdly named website.

**Reply by ReaLKing** [reply]
It was an example- there was nothing fishy meant by it @LaneSh4d0w - was referring mainly to the beginning the plugin://  as @Sphinxroot and I were trying to see how to convert correctly to an http/https or rtmp... Apologies for the bad example- if you look at the 2nd example i had sent I'm sure you can see that was trying to refer to the plugin:// - Thank you

**Reply by BellezaEmporium** [reply]
> It was an example- there was nothing fishy meant by it @LaneSh4d0w - was referring mainly to the beginning the plugin://  as @Sphinxroot and I were trying to see how to convert correctly to an http/https or rtmp... Apologies for the bad example- if you look at the 2nd example i had sent I'm sure you can see that was trying to refer to the plugin:// - Thank you   

Do not worry, I was only referring to the way the website worked, which could be the reason why Streamlink didn't really get it.

**Reply by ReaLKing** [reply]
Understood- Thank you- sorry for the confusion.
